### PR TITLE
Rename Replaceable Monitoring Cluster Definition XML

### LINKED ISSUE #26980: Rename Replacable Monitoring Cluster
# 📖 Description
Rename `replacable-monitoring-cluster.xml` to `resource-monitoring-cluster.xml`

## 🔎 Details
The cluster name is going to be changed by [this PR](https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7115) because of [this issue](https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/6780)

### PR DESCRIPTION
**Pull request by p0fi**
# 📖 Description
This PR renames the cluster definition file from `replacable-monitoring-cluster.xml` to `resource-monitoring-cluster.xml`

## 🔗 Links/References
- fixes #26980

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -21,7 +21,8 @@ on:
     workflow_dispatch:
 
 concurrency:
-    group: ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name ==
+    group:
+        ${{ github.ref }}-${{ github.workflow }}-${{ (github.event_name ==
```
Comment (bzbarsky-apple) [inline .github/workflows/tests.yaml:25]: Why this unrelated change?

File: .github/workflows/tests.yaml [hunk 2]
```diff
@@ -43,15 +44,18 @@ jobs:
         env:
             BUILD_VARIANT: ${{matrix.build_variant}}
             CHIP_TOOL_VARIANT: ${{matrix.chip_tool}}
-            TSAN_OPTIONS: "halt_on_error=1 suppressions=scripts/tests/chiptest/tsan-linux-suppressions.txt"
+            TSAN_OPTIONS:
+                "halt_on_error=1
+                suppressions=scripts/tests/chiptest/tsan-linux-suppressions.txt"
```
Comment (bzbarsky-apple) [inline .github/workflows/tests.yaml:49]: And these?  It looks like you have some auto-formatter running that has different opinions from the existing formatting.  Please get rid of the spurious changes, so this can be reviewed sanely without minutely double-checking all the thigns that got munged.

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yaml (hunk 2 above)
Comment (p0fi) [reply .github/workflows/tests.yaml:49]: Oh sorry! Those changes sneaked in. It's because there is a prettier config in the project (see [here](https://github.com/project-chip/connectedhomeip/blob/master/.prettierrc.json)) and it's applied to this file. Which actually seems reasonable since there is such config and im wondering why this is not applied for others too 😅

Comment (bzbarsky-apple) [reply .github/workflows/tests.yaml:49]: I have no idea what that prettier config is, but clearly it's not used by the people who are normally editing these files....  ;)

**Review by bzbarsky-apple — APPROVED** [review]

File: .github/workflows/tests.yaml (hunk 2 above)
Comment (p0fi) [reply .github/workflows/tests.yaml]: Prettier is a code formatter tool which is pretty awesome and its in the project since 3 years :D 
That being said pushed revert of those changes 🙂

**Review by yufengwangca — APPROVED** [review]
